test/runtests.jl

```diff
@@ -223,11 +223,12 @@ end
     dpar[:var] = MD(0.0, Dict{Symbol, MD}())
     dpar[:sub] = MD(0.0, Dict{Symbol, MD}(:a=>MD(0.0, Dict{Symbol, MD}())))
 
-    autodiff(sum_rec, Duplicated(par, dpar))
-    @show par, dpar, sum_rec(par)
-    @test dpar[:var].v ≈ 1.0 
-    @test dpar[:sub].v ≈ 1.0 
-    @test dpar[:sub].d[:a].v ≈ 1.0 
+    # TODO
+    # autodiff(sum_rec, Duplicated(par, dpar))
+    # @show par, dpar, sum_rec(par)
+    # @test dpar[:var].v ≈ 1.0 
```

Comment:
Can we mark this `@test_broken` or do we crash?

Reply:
Nope, crash (at least historically)